packages/l/libmem/xmake.lua

```diff
@@ -0,0 +1,39 @@
+package("libmem")
+
+    set_homepage("https://github.com/rdbo/libmem")
+    set_description("Cross-platform game hacking library for C, C++, Rust, and Python, supporting process/memory hacking, hooking, detouring, and DLL/SO injection.")
+    set_license("AGPL-3.0")
+
+    add_urls("https://github.com/rdbo/libmem.git", {submodules = true})
+    --add_urls("https://github.com/rdbo/libmem/archive/refs/tags/$(version).tar.gz", {submodules = true}) 
+    add_versions("5.0.2", "99adea3e86bd3b83985dce9076adda16968646ebd9d9316c9f57e6854aeeab9c")
+    add_deps("cmake", "vcpkg::capstone", "vcpkg::keystone") -- "vcpkg::llvm 11.1.0"
```

Comment:
please do not add 3rd deps, we should use packages in xmake-repo.

Reply:
Is it a day when I need port keystone over xrepo?

Reply:
Done, I have made my own keystone port.